### PR TITLE
Feature/auto create destroy webhooks

### PR DESCRIPTION
**Pull request by TarCode**
Hey Josh, can you review the create part for me please? Will do the update once this is cool.

### CODE REVIEW

**Comment by TarCode** [comment]
NB: I added BASE_URL to .env

File: src/config/plugins/urls.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import os
+
+BASE_URL = os.environ.get('BASE_URL')
```
Comment (TarCode) [inline src/config/plugins/urls.py]: @OdinsPlasmaRifle Here's where BASE_URL gets loaded via os.environ

**Comment by TarCode** [comment]
Changes have been made, update method has been added.

File: src/notifications/serializers.py [hunk 2]
```diff
@@ -218,20 +218,139 @@ def validate(self, validated_data):
...
+            event = validated_data['event']
+
+            # Make sure the event is changing values.
+            if event !== instance.event:
+                # Check if the current event is required. Excluding this
+                # instance.
+                is_event_required = Notification.objects.filter(
+                    company=instance.company,
+                    event=instance.event
+                ).exclude(id=instance.id)exists():
```
Comment (ConnorMac) [inline src/notifications/serializers.py]: Pretty sure we need a `.` here

Comment (OdinsPlasmaRifle) [reply src/notifications/serializers.py]: You are correct :-P

**Pull request merged**